applications/nginx-search/src/main/docker/nginx.conf

```diff
@@ -114,6 +114,12 @@ http {
             proxy_pass http://registration-api:8080;
         }
 
+        location /metrics/apis {
+            rewrite /metrics/apis/liveness /actuator/health break;
```

Comment:
Hvorfor kaller du helsesjekkene for /actuator internt ? (Er ikke nødvendigvis uenig, men nysgjerrig på tanken bak).